frontend/src/pages/learn/challenge-solution/[moduleId].tsx

```
@@ -0,0 +1,82 @@
+
+    useEffect(() => {
+        fetchContent();
+    }, [moduleId]);
+
+    if (!content) {
+        return <p>Loading...</p>;
+    }
+
+    return (
+        <div className={styles.container}>
+            <div dangerouslySetInnerHTML={{ __html: content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)